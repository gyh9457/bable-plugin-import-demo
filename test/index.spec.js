import { transformSync, createConfigItem } from '@babel/core'
import plugin from '../src'

const example = `import { A, B, C as D } from 'foo'`

it('plugin test', () => {
  const { code } = transformSync(example, {
    plugins: [[plugin, { libraryName: 'foo' }]]
  })
  expect(code).toMatchSnapshot()
}) 