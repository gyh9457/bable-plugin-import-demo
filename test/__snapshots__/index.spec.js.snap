// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`plugin test 1`] = `
"\\"use strict\\";

var _A = _interopRequireDefault(require(\\"foo/lib/A\\"));

require(\\"foo/lib/A/style.css\\");

var _B = _interopRequireDefault(require(\\"foo/lib/B\\"));

require(\\"foo/lib/B/style.css\\");

var _C = _interopRequireDefault(require(\\"foo/lib/C\\"));

require(\\"foo/lib/C/style.css\\");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }"
`;
